signup/signup.css

```diff
@@ -0,0 +1,64 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.login-container{
+    display: flex;
+    height: 80vh;
+    align-items: center;
+    justify-content: center;
+}
+
+.login-title{
+    text-align: center;
+}
+.input-container{
+    margin: 2rem 0rem;
+}
+
+.input-container a{
+    color: inherit;
+    text-decoration: none;
+}
+
+#conditions{
+    display: inline-flex;
+    font-size: 0.8rem;
+    align-items: center;
+}
+
+input[type="checkbox"]{
+    margin-right: 0.5rem;
+    width: 0.8rem;
+    height: 0.8rem;
+}
+
+.input-primary{
+    border-radius: 0.2rem;
+    padding: 0.5rem;        
+}
+
+.login-card{
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 6px -1px, rgba(0, 0, 0, 0.06) 0px 2px 4px -1px;
+    width: 22rem;
+    padding: 0.5rem;
+    border: 2px solid rgba(0, 0, 0, 0.1);
+}
+
+#userName{
```

Comment:
try to use class instead of id.

Reply:
I know id are less preferred. But this style is specific to those only. I don't need them anywhere else.

Reply:
ok no issue then